### PR TITLE
[FIXED JENKINS-42245] Add description according to Wiki

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-42245](https://issues.jenkins-ci.org/browse/JENKINS-42245)

@reviewbybees @kohsuke @fbelzunc

### CODE REVIEW

**Comment by recena** [comment]
I love the details.

File: pom.xml [hunk 1]
```diff
@@ -107,14 +108,14 @@
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
```
Comment (aheritier) [inline pom.xml]: Should be in a different commit

File: pom.xml [hunk 2]
```diff
@@ -107,14 +108,14 @@
...
     </repository>
   </repositories>
 
   <pluginRepositories>
     <pluginRepository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
```
Comment (aheritier) [inline pom.xml]: Should be in a different commit

File: pom.xml (hunk 1 above)
Comment (varyvol) [inline pom.xml]: Fair enough. Removed and then added in another commit.

**Comment by varyvol** [comment]
@aheritier could you kindly review once the changes have been applied?

**Comment by aheritier** [comment]
Could you rewrite the history @varyvol please ? To squash commits 1 and 2 together. After that it will be perfect. thx

**Comment by jglick** [comment]
Squashing can be done at merge time if you require this for some reason. PR submitters should never squash their own commits unless they have divulged nuclear launch codes or something.
